ompi/mca/hook/demo/configure.m4

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2016      IBM Corporation.  All rights reserved.
```

Comment:
2017.

Should probably update all copyrights to 2017, because that is when it was introduced into the Open MPI community code base.